extensions/amp-story/1.0/test/test-amp-story.js

```diff
@@ -755,6 +755,23 @@ describes.realWin('amp-story', {
                 .to.be.false;
           });
     });
+
+    it('should close sidebar mask with close action', () => {
+      createPages(story.element, 2, ['cover', 'page-1']);
+
+      const sidebar = win.document.createElement('amp-sidebar');
+      story.element.appendChild(sidebar);
+
+      story.buildCallback();
+      return story.layoutCallback()
+          .then(() => {
+            story.storeService_.dispatch(Action.TOGGLE_SIDEBAR, false);
+          }).then(() => {
+            const mask = story.element.querySelector(
+                '.i-amphtml-story-opacity-mask');
+            expect(mask.hasAttribute('hidden')).to.be.true;
```

Comment:
Nit: `expect(mask).to.have.attribute('hidden');` gives cool error messages :))

Reply:
Got it! Thanks.